ui.py

```diff
@@ -1,25 +1,26 @@
-#!/usr/bin/env python3
-# -*- coding: utf-8 -*-
+#! /usr/bin/env python3
+# coding=utf-8
+
 
 import os
 import sys
 
-# If we're on Windows, use the included compiled DLLs.
-if sys.platform == "win32":
-    os.environ["PYSDL2_DLL_PATH"] = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'libs')
-
-from sdl2 import *
-from sdl2.sdlttf import *
 import sdl2.ext
 
-from const import Colors, WindowSize
-
+from sdl2 import *  # Bad idea to import like this!
```

Comment:
Hi
For this to load on windows "PYSDL2_DLL_PATH" doens't have to go first?
thanks